app/lib/theme/widgets/SelectWithNofifyWidget.tsx

```diff
@@ -0,0 +1,69 @@
+import { WidgetProps } from "@rjsf/core";
+import Dropdown from "@button-inc/bcgov-theme/Dropdown";
+import { Button } from "@button-inc/bcgov-theme";
+
+interface Option {
+  type: string;
+  title: string;
+  enum: number[];
+  value: number;
+}
+
+const SelectWithNotifyWidget: React.FunctionComponent<WidgetProps> = (
+  props
+) => {
+  const {
+    id,
+    onChange,
+    schema,
+    placeholder,
+    label,
+    required,
+    uiSchema,
+    value,
+  } = props;
+
+  if (!(schema && schema.anyOf && typeof schema.anyOf !== "undefined")) {
+    throw new Error("schema.anyOf does not exist!");
+  }
```

Comment:
This is similar to both `SelectWidget` and https://github.com/bcgov/cas-cif/pull/1233/files#diff-c25c21c07939825b806d9f458429c946e6791267f81751714abf5893c38f904f 
As we discussed we should maybe open a tech debt to keep these components DRY.

Reply:
Agreed, this schema check could be broken out into a reusable function since a lot of rjsf widgets seem to use it. Calling `anyOf` ensures that [at least one of the subschemas is valid.](https://react-jsonschema-form.readthedocs.io/en/docs/usage/oneof/)